tools/topology/topology2/include/components/gain.conf

```diff
@@ -171,4 +183,5 @@ Class.Widget."gain" {
 	init_value		0x7fffffff
 	num_input_pins		1
 	num_output_pins		1
+	name_no_prefix		"false"
```

Comment:
not following, the first part of this patch adds 'use_topology_name' but this sets 'name_no_prefix'

I don't see the link between the two.

Reply:
The both mean "do not add the widget name as a prefix to the kcontrol name that is visible in the alsa mixer, e.g. use the topology name only". However, on the Linux side referring to topology is not very accurate as the widget may not be coming from a topology file at all, but from some ASoC component driver for instance.

Reply:
Now I am also lost, we have both name_no_prefix and use_topology_name, but they are the same concept?

Why can't we have just one token added?

Reply:
Ok, I go back to name_no_prefix everywhere.

Reply:
@jsarha shouldn't this be a mixer control attribute instead of a gain? WHat happens if different component has a mixer control?